src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -288,6 +288,10 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<_NativeAssemblySourceDir>$(IntermediateOutputPath)android\</_NativeAssemblySourceDir>
 	<_AndroidUseNewTypemaps>True</_AndroidUseNewTypemaps>
 	<_SkipJniAddNativeMethodRegistrationAttributeScan Condition=" '$(_SkipJniAddNativeMethodRegistrationAttributeScan)' == '' ">False</_SkipJniAddNativeMethodRegistrationAttributeScan>
+
+	<_AndroidFastDeployEnvironmentFiles Condition=" '$(_AndroidFastDeployEnvironmentFiles)' == '' And '$(EmbedAssembliesIntoApk)' == 'true' ">True</_AndroidFastDeployEnvironmentFiles>
```

Comment:
@dellis1972: This `Condition` feels backwards to me? `$(_AndroidFastDeployEnvironmentFiles)` reads as "affirmative", as in "if `$(_AndroidFastDeployEnvironmentFiles)` is true, then `@(AndroidEnvironment)`/etc. should *also* be fast deployed".

Yet, `$(_AndroidFastDeployEnvironmentFiles)` will only default to True if `$(EmbedAssembliesIntoApk)` is true, which means fast deployment is *disabled*.

I don't think I understand the semantics of this property. :-(